### DISCUSSION TITLE
Quarkus Mailer - Attachments not Being Sent in Mail

### DISCUSSION THREAD

**Discussion by pmsmm** [quarkusio/quarkus#42673] [Q&A] [answered]
Hello everyone, first time posting here and a very very recent Quarkus user so I hope I'm not doing anything wrong and can count on your help!

My issue is related to the Quarkus Mailer not sending attachments as part of an email when running the code via the JAR. The question can be found [here](https://stackoverflow.com/questions/78893108/quarkus-mailer-not-inserting-attachments-when-sending-email).

Long story short because I know the Stack Overflow post is quite big, when I try sending attachments in my email they are not sent and no errors are reported but when testing the same code I can see that the attachments are part of the email, the problem only happens when actually sending the email by running the code through the JAR.

I don't know if this could actually be related to the Quarkus Mailer libraries but I feel like there should be a log that somehow states why the email is sent without the attachments because as of now it is a silent problem.

[a comment by an automated account is not shown]

**Reply by gsmet** [reply]
I posted an answer on SO. Let's continue there as we have all the context.

**Comment by pmsmm** [comment, marked as answer]
For anyone interested on the underlying issue please check the SO thread, I posted the answer there. Thanks again to @gsmet for your support!
